stable/kommander/templates/grafana/ingress-roles.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.portalRBAC.grafana.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: opsportal-kommander-monitoring-thanos-admin
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+rules:
+  - nonResourceURLs:
+      - {{ .Values.grafana.ingress.path | trimSuffix "/"}}
+      - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/*
+    verbs:
+      - "*"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: opsportal-kommander-monitoring-thanos-view
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+rules:
+  - nonResourceURLs:
+      - {{ .Values.grafana.ingress.path | trimSuffix "/"}}
+      - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/*
+    verbs:
+      - get
+      - head
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: opsportal-kommander-monitoring-thanos-edit
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+rules:
+  - nonResourceURLs:
+      - {{ .Values.grafana.ingress.path | trimSuffix "/"}}
+      - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/*
+    verbs:
+      - get
+      - head
+      - post
+      - put
+      - patch
+  {{- end}}
```

Comment:
nit: should this line be unindented? same q for the others

Reply:
:+1:

Reply:
looks like this is still indented

Reply:
I don't know how I missed that